packages/compass-smoke-tests/src/build-info.ts

```diff
@@ -25,13 +27,21 @@ function assertObjectHasKeys(
 
 // subsets of the hadron-build info result
 
-export const commonKeys = ['productName'] as const;
-export type CommonBuildInfo = Record<typeof commonKeys[number], string>;
+export const commonKeys = ['productName', 'channel'] as const;
+export type CommonBuildInfo = Record<typeof commonKeys[number], string> & {
+  channel: 'dev' | 'beta' | 'stable';
```

Comment:
This could more easily be derived from `SUPPORTED_CHANNELS` (see the comment declaring `Channel` above).

```suggestion
  channel: Channel;
```